### PR TITLE
README Quickstart Sync 

### PR DESCRIPTION
**Pull request by JacobMGEvans**
- Format aligns with new and up to date Quickstarts (QS) 
- Banner and socials match other QS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,105 @@
 <p align="center">
-  <a href="https://www.clerk.com/?utm_source=github&utm_medium=starter_repos&utm_campaign=clerk_remix_v2" target="_blank" align="center">
```
Comment (JacobMGEvans) [inline README.md:2]: Should I keep the UTM even if we are changing the name? I used the clerk-docs UTM from the other QuickStarts for that link

Comment (royanger) [reply README.md:2]: Change the UTM to `clerk_remix_quickstart`

Comment (JacobMGEvans) [reply README.md:2]: Done! Matches the NextJS repos convention for this now, thanks 😄

**Review by royanger — APPROVED** [review]
🚀

**Pull request merged**